.ci/azure-pipelines/build_linux.bash

```diff
@@ -1,26 +1,40 @@
 #!/bin/bash -ex
 #
-# Copyright 2019-2020 The Mumble Developers. All rights reserved.
+# Copyright 2019-2021 The Mumble Developers. All rights reserved.
```

Comment:
This is an unrelated change. We should update the copyright headers for _all files_ in a _separate commit_ :point_up:

Reply:
I agree.